elastic/apm-server
# aggregation/txmetrics: group by `service.node.name`, `cloud.{provider,region,availability_zone}`

## Motivation/summary

Update transaction metrics aggregation keys to include:
 - `service.node.name`: see https://github.com/elastic/apm-server/issues/6315
 - `cloud.{provider,region,availability_zone}`: see https://github.com/elastic/apm-server/issues/6099

## Checklist

~- [ ] Documentation has been updated~

## How to test these changes

1. Send transactions to apm-server with service node name explicitly set (`ELASTIC_APM_SERVICE_NODE_NAME`)
2. Ensure aggregated metrics include the service node name

## Related issues

Closes https://github.com/elastic/apm-server/issues/6315
Closes #6099
